ci/buildkite.py

```diff
@@ -139,10 +143,35 @@ def cmd_pipeline(args, remainder):
         yml = pystache.render(load_template('pipeline.yml'), ctx)
         print(yml)
 
+
+def wheel_path(arg):
+    if platform.system() == 'Windows':
```

Comment:
Instead of using the `bazel-bin` symlink, I think we should be using `bazel info bazel-bin` to get the absolute path.